Fix access om2 issue 904 (#905)
ACCESS-NRI/access-hive.org.au: docs/models/run-a-model/run-access-om.md

If there are no jobs listed with your `jobname` (or if no job is listed), your run either successfully completed or was terminated due to an error.<br>
For more information, check [NCI documentation](https://opus.nci.org.au/display/Help/FAQ+1%3A+Why+My+Jobs+are+NOT+Running).

!!! tip

>     While the model is running, you can get an idea of where it's up to with

Reviewer: ```suggestion
    While the model is running, you can monitor its progress by running:
```
